.github/workflows/k8s-kind-network-policies-e2e.yaml

```diff
@@ -203,6 +203,17 @@ jobs:
             --report-dir=${E2E_REPORT_DIR}                \
             --disable-log-dump=true
 
+      - name: Poor man's file-descriptor leak detector
```

Comment:
I think it might be useful to add some context here since this check is randomly in the netpol e2e workflow.